deployments/helm-chart/values.yaml

```diff
@@ -178,6 +178,9 @@ controller:
   ## Enable OIDC policies.
   enableOIDC: false
 
+  ## Include year in log header. This parameter will be removed in release 2.7 and the year will be included by default.
+  includeYear: false
```

Comment:
a corresponding entry in https://github.com/nginxinc/kubernetes-ingress/blob/feature/flag-include-year/docs/content/installation/installation-with-helm.md  as `controller.includeYear`

Reply:
Not sure I understand this. None of the values in this file start with `controller.` why would the `includeYear` value need to start with that?

Reply:
it refers to https://github.com/nginxinc/kubernetes-ingress/blob/feature/flag-include-year/deployments/helm-chart/values.yaml#L1 when you're installing using ` helm install nginx-ingress nginx-stable/nginx-ingress --set controller.kind=daemonset --set controller.includeYear=true ......`